pandas/core/groupby/groupby.py

```diff
@@ -5715,10 +5716,17 @@ def _idxmax_idxmin(
         )
         return result
 
-    def _wrap_idxmax_idxmin(self, res: NDFrameT) -> NDFrameT:
+    def _wrap_idxmax_idxmin(
+        self, res: NDFrameT, how: Literal["idxmax", "idxmin"], skipna: bool
+    ) -> NDFrameT:
         index = self.obj.index
         if res.size == 0:
             result = res.astype(index.dtype)
+        elif skipna and res.lt(0).any(axis=None):
```

Comment:
Could `res.lt(0).any(axis=None)` ever return a missing value that would raise if used in a `bool` context like this?